.vscode/settings.json

```diff
@@ -78,6 +78,16 @@
       "username": "dataset_service",
       "password": "changeme"
     },
+    {
+      "previewLimit": 50,
+      "server": "localhost",
```

Comment:
@tschaffter
The database only works when I used `localhost` for server, but not using project name `bixarena-postgres` like other postgres projects. Do you know why?

Reply:
This script is triggered when opening a new terminal.